plugins/MidiExport/MidiFile.hpp

```diff
@@ -239,10 +239,15 @@ class MIDITrack
 		addEvent(event);
 	}
 	
-	inline void addTempo(uint8_t tempo, uint32_t time)
+	inline void addTempo(uint32_t tempo, uint32_t time)
 	{
-		Event event; event.channel = channel;
-		event.type = Event::TEMPO; event.time=time; event.tempo = tempo;
+		Event event;
+		event.channel = channel;
+
+		event.type = Event::TEMPO;
+		event.time=time;
```

Comment:
If you're going to fix the style, why don't you fix this?
```suggestion
		event.time = time;
```

Reply:
I missed it, I guess I only focused on the obvious of "Hey these are all on one line and they shouldn't be"